Dockerfile

```diff
@@ -25,15 +25,17 @@ ENV DOCKER_ENABLE_SECURITY=false \
 RUN echo "@testing https://dl-cdn.alpinelinux.org/alpine/edge/main" | tee -a /etc/apk/repositories && \
     echo "@testing https://dl-cdn.alpinelinux.org/alpine/edge/community" | tee -a /etc/apk/repositories && \
     echo "@testing https://dl-cdn.alpinelinux.org/alpine/edge/testing" | tee -a /etc/apk/repositories && \
+    apk update && \
```

Comment:
why are downloading the cache, never use it and then delete it at the end again?

Reply:
Good point, will remove this on next branch

Reply:
just want to remind you that it is still no changed

Reply:
just want to remind you that it is still no changed